packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpUssApi.ts

```diff
@@ -88,6 +88,19 @@ export class FtpUssApi extends AbstractFtpApi implements MainframeInteraction.IU
         }
     }
 
+    public async uploadFromBuffer(buffer: Buffer, filePath: string, options?: zowe.IUploadOptions): Promise<zowe.IZosFilesResponse> {
```

Comment:
Seems like the implementation is similar to `ZoweExplorerFtpMvsApi.uploadFromBuffer` - perhaps we could add a utility method to reduce code duplication? I also think its ok to leave as-is for now until we have to reuse this code a 3rd time somewhere else 😋

Reply:
I'll add a utility function in the FileSystemProvider PR - hoping I can generalize it enough that it can be added to the API and used by extenders if they don't have an easy way to upload from buffer.